pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExporter.java

```diff
@@ -47,20 +56,67 @@
 
     // A cache for user auth0 emails, storing (auth0UserId -> email).
     private static final Map<String, String> emailStore = new HashMap<>();
+    private static final String familyIdHeader = "familyId";
 
-    private final Config cfg;
+    private final Config mainConfig;
+    private final Config exportConfig;
     private final String customGuid;
     private final String customActivity;
 
-    public CustomExporter(Config cfg) {
-        this.cfg = cfg;
-        this.customActivity = cfg.getString(CustomExportConfigFile.ACTIVITY);
-        this.customGuid = cfg.getString(CustomExportConfigFile.STUDY_GUID);
+    public CustomExporter(Config mainConfig, Config exportConfig) {
+        this.mainConfig = mainConfig;
+        this.exportConfig = exportConfig;
+        this.customActivity = exportConfig.getString(CustomExportConfigFile.ACTIVITY);
+        this.customGuid = exportConfig.getString(CustomExportConfigFile.STUDY_GUID);
     }
 
+    List<CustomExportParticipant> extractParticipantDataSetByIds(Handle handle, StudyDto studyDto, Set<Long> userIds)
+            throws IOException {
```

Comment:
I don't see where `IOException` is coming from. It looks like we're wrapping it in an unchecked exception already.